alembic/env.py

```diff
@@ -0,0 +1,83 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from city.models import *
```

Comment:
Importing all from a module using `from city.models import *` is generally discouraged as it can lead to namespace conflicts and makes it harder to identify where specific classes or functions are coming from. It's better to import only the necessary components explicitly.